submission_rules.adoc

```diff
@@ -237,6 +237,7 @@ Some working groups such as HPC may choose to replace the Advisory Board formati
 | T-20 weeks | Advisory Board signs off, Model Frozen, Working groups sync on final benchmark. Finishing touches on benchmark can commence.
 | T-16 weeks | Benchmark code complete. Only bug fixes allowed beyond this point.
 | T-12 weeks | RCPs due (for Training and HPC)
+| T-9 weeks | New RCP requests due (for LLM Training only)
```

Comment:
What's the relationship between this line and the line above? Looks confusing to me.

Reply:
what is the new RCP request deadline for the rest of the benchmarks?

Reply:
> What's the relationship between this line and the line above? Looks confusing to me.

I shall remove this line because the timeline for LLM RCP requests is not related to this table (which only has the timeline for new benchmarks).

> what is the new RCP request deadline for the rest of the benchmarks?

For other benchmarks, a submitter can provide RCPs until submission time. This is not the case for LLM though.